bob-backend/src/pearl/data.rs

```diff
@@ -35,25 +42,47 @@ impl AsRef<Key> for Key {
     }
 }
 
+impl<'a> From<&'a [u8]> for RefKey<'a> {
+    fn from(v: &'a [u8]) -> Self {
+        Self(v)
+    }
+}
+
+fn le_cmp(x: &[u8], y: &[u8]) -> std::cmp::Ordering {
```

Comment:
Since this function is outside of `Key` and `RefKey`, it would be good to indicate in its name that it is for key comparison (e.g. `le_cmp_keys`)

Reply:
Done